packages/admin/admin-color-picker/src/ColorPicker.tsx

```diff
@@ -0,0 +1,183 @@
+import { InputWithPopper, InputWithPopperComponents, InputWithPopperComponentsProps, InputWithPopperProps } from "@comet/admin";
+import { Box, ComponentsOverrides, InputAdornment, InputBaseProps, Theme } from "@mui/material";
+import { WithStyles, withStyles } from "@mui/styles";
+import * as React from "react";
+import { HexColorPicker, RgbaStringColorPicker } from "react-colorful";
+import { ColorPickerBaseProps } from "react-colorful/dist/types";
+import tinycolor from "tinycolor2";
+import { useDebouncedCallback } from "use-debounce";
+
+import { ColorPickerClassKey, styles } from "./ColorPicker.styles";
+
+export interface ColorPickerPropsComponents extends InputWithPopperComponents {
+    ColorPickerColorPreview?: React.ElementType<ColorPickerColorPreviewProps>;
+    ColorPickerInvalidPreview?: React.ElementType<React.HTMLAttributes<HTMLDivElement>>;
+    ColorPickerEmptyPreview?: React.ElementType<React.HTMLAttributes<HTMLDivElement>>;
+}
+
+export interface ColorPickerPropsComponentsProps extends InputWithPopperComponentsProps {
+    hexColorPicker?: Partial<ColorPickerBaseProps<string>>;
+    rgbaStringColorPicker?: Partial<ColorPickerBaseProps<string>>;
+}
+
+export interface ColorPickerProps extends Omit<InputWithPopperProps, "children" | "onChange" | "value" | "componentsProps" | "components"> {
+    value?: string | null;
+    onChange?: (color: string | null) => void;
+    colorFormat?: "hex" | "rgba";
+    colorPalette?: string[];
+    hidePicker?: boolean;
```

Comment:
Maybe we should call this prop `disablePicker` to be more in line with MUI

Reply:
I hat it that way, but changed it, because I didn't want it to be confused with disabling the whole DatePicker 😄